api/envoy/config/transport_socket/tap/v2alpha/tap.proto

```diff
@@ -6,13 +6,15 @@ import "envoy/api/v2/core/base.proto";
 import "envoy/config/common/tap/v2alpha/common.proto";
 
 import "udpa/annotations/migrate.proto";
+import "udpa/annotations/status.proto";
 import "validate/validate.proto";
 
 option java_package = "io.envoyproxy.envoy.config.transport_socket.tap.v2alpha";
 option java_outer_classname = "TapProto";
 option java_multiple_files = true;
 option (udpa.annotations.file_migrate).move_to_package =
     "envoy.extensions.transport_sockets.tap.v3";
```

Comment:
We can use this annotation altogether with the status, we can easily achieve `v3alpha` package for files subject to breaking change.